tests/PHPStan/Rules/Functions/data/fputcsv-fields-parameter-php8.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+class StringablePerson implements \Stringable {
```

Comment:
Please don't put these test files in global namespace. Some namespace like "namespace FputcsvPhp8" would work.

Reply:
Added